packages/cli/src/commands/deploy/aws.js

```diff
@@ -68,28 +82,34 @@ export const handler = async ({ provider }) => {
         title: 'Building and Packaging...',
         task: () =>
           new Listr(
-            providerData.buildCommands.map((commandDetail) => {
-              return {
-                title: commandDetail.title,
+            [
+              {
+                title: providerData.buildCommands[0].title,
                 task: async () => {
-                  await execa(...commandDetail.command, {
+                  await execa(...providerData.buildCommands[0].command, {
                     cwd: BASE_DIR,
                   })
                 },
-              }
-            }),
+              },
+              {
+                title: 'packing',
+                task: pack,
+              },
+            ],
```

Comment:
Definitely a little messy getting the first element of the array only, I didn't worry too much about making this beautiful since it relates to this question https://github.com/redwoodjs/redwood/pull/3782/files#r757742798

How should this be setup now that it's not using a cli command?

Reply:
I think maybe it would be better to get the command in a step above the lister?

e.g.
```js
const serverlessListrSteps = mapCommandsToListr(providerData.buildCommands)
```

Reply:
I wasn't completely sure what you wanted here, does 9b8192c hit the mark?